test/e2e/kubevirt.go

```diff
@@ -1649,6 +1659,41 @@ runcmd:
 				Eventually(clusterUserDefinedNetworkReadyFunc(fr.DynamicClient, cudn.Name), 5*time.Second, time.Second).Should(gomega.Succeed())
 				udnName = cudn.Name
 				networkName = util.GenerateCUDNNetworkName(cudn.Name)
+				if td.ingress == "routed" {
+					ginkgo.By("create router advertisement")
+					raClient, err := raclientset.NewForConfig(fr.ClientConfig())
+					Expect(err).NotTo(HaveOccurred())
+
+					raApplyCfg := applycfgrav1.RouteAdvertisements(udnName).
```

Comment:
Super NIT: with the latest API  you can just create the object, no need for apply configuration. It doesn't hurt though.